lib/allowed_organisations/local_authorities.yml

```diff
@@ -1,3 +1,5 @@
 ---
 - '309'
 - '330'
+- '925'
```

Comment:
we tend to sort things alphabetically or numerally in this case for easier look-up

Reply:
ok thanks! good to know